### DISCUSSION TITLE
How is Grids Own Consumption Considered in CWatM

### DISCUSSION THREAD

**Discussion by Zprivf** [iiasa/CWatM#62] [Q&A] [answered]
Hello @guillaumot, 
This may be basic, but I want to ensure I understand correctly how demand is considered in CWatM? 
Does the resulting grid discharge (output) consider grid consumption? Or is the grid’s own consumption subtracted when routed to the next grid (which I see in the routing code)? 
Qgrid= WA-GridsOwnConsumption
Or 
Qgrid= WA
Where WA is available water.

Thank you

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Tag @Zprivf,
Grid cell discharge is reduced by its consumption.
Qgrid = runoff - channel evaporation - channel abstraction + return flow (all grid cell specific)

**Reply by Zprivf** [reply]
Thank you @mikhailsmilovic
